core/templates/dev/head/pages/exploration_editor/editor_tab/state_editor_interaction.html

```diff
@@ -36,7 +36,7 @@ <h3 ng-if="!stateInteractionIdService.displayed">
       Choose Interaction
     </h3>
 
-    <h3 ng-if="stateInteractionIdService.displayed">
+    <h3 ng-if="stateInteractionIdService.displayed && customizationModalReopened">
```

Comment:
You might need to double check the logic. I poked around a bit at this and am finding cases where the header doesn't show after I pick a new interaction, so I get confused when coming to this screen since there's no context. 

![oops](https://cloud.githubusercontent.com/assets/10575562/23992284/8eb8bb52-09f9-11e7-93fe-badd9f34d166.png)

Could you please stress-test your code and try to break it? We should fix all these corner cases before submit.